components/image-releaser/app.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: 0.1.0
+environments:
+  default:
```

Comment:
nit: get rid of default environment as well since both releasing and default env point to the same server and namespace